### PR TITLE
#24 feature: fetch cart Items

### PR DESCRIPTION
**Pull request by Muhet**
#### What does this PR do?
View cart item

#### Description of Task to be completed?

Being able to view the contents of the cart is an essential functionality for any e-commerce website, as it allows buyers to review their selections and proceed to checkout.

#### How should this be manually tested?
```
git clone https://github.com/atlp-rwanda/ecommerce-app-predators-fe.git
cd ecommerce-app-predators-fe
git checkout ft-view-cart-item-24
npm i
npm run dev
```

#### What are the relevant pivotal tracker/Trello stories?
Trello [Link]() (#24)

#### Screenshots (if appropriate)
Predators

### CODE REVIEW

**Review by yobuir — COMMENTED** [review]
resolve the conflicts

**Review by nili-alu — APPROVED** [review]

**Review by yobuir — APPROVED** [review]

File: src/components/ProductListing.tsx [hunk 1]
```diff
@@ -1,33 +1,16 @@
...
-  available: boolean;
-  vendor_id: number;
-  createdAt: string;
-  updatedAt: string;
-}
-
 function ProductListing() {
   const products = useSelector(
-    (state: { products: { data: Product[] } }) => state.products.data
+    (state: { products: { data: any } }) => state.products.data
```
Comment (chanceDira) [inline src/components/ProductListing.tsx]: replace these any with interfaces

File: src/components/ProductListing.tsx [hunk 2]
```diff
@@ -45,16 +28,16 @@ function ProductListing() {
...
-        <div className="flex" key={product.id}>
-          <h2>{product.name}</h2>
-          <p>{product.id}</p>
-          <p>{product.picture_urls[2]}</p>
-
-        </div>
-      ))}
+      {products?.data?.products
+        .slice(0, 1)
+        .map((product: { id: any; name: any; description: any }) => (
```
Comment (chanceDira) [inline src/components/ProductListing.tsx]: no any(s)

**Review by chanceDira — CHANGES_REQUESTED** [review]
Below are some requested changes, Also initially the total price is set to $23.20, you can fix that

**Review by vanessa-abewe — APPROVED** [review]

**Review by chanceDira — APPROVED** [review]
Looking good

**Pull request merged**